package.json

```diff
@@ -26,10 +26,11 @@
     "pages:build": "cross-env NODE_ENV=production NITRO_PRESET=cloudflare_pages nuxt build --standalone"
   },
   "devDependencies": {
-    "@erc725/erc725.js": "^0.18.0",
+    "@erc725/erc725.js": "~0.17.1",
```

Comment:
v0.18 is not working?

Reply:
Yes it didn't work with my older profiles which I'm using for testing. I will bump up everything to latest versions later.